### PR TITLE
QPPCT-743: ignore NPI for non individual submissions

### PR DESCRIPTION
**Pull request by clydet**
### Information
-  QPPCT-743

### Changes proposed in this PR:
- ignore NPI for non individual submissions

### Checklist 
- [ ] All JUnit tests pass (`mvn clean verify`).
- [ ] New unit tests written to cover new functionality.
- [ ] Added and updated JavaDocs for non-test classes and methods.
- [ ] No local design debt. Do you feel that something is "ugly" after your changes?
- [ ] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

**Comment by clydet** [comment]
word

File: converter/src/main/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoder.java [hunk 1]
```diff
@@ -51,7 +51,9 @@ protected DecodeResult decode(Element element, Node thisNode) {
 		setProgramNameOnNode(element, thisNode);
 		setEntityIdOnNode(element, thisNode);
 		setPracticeSiteAddress(element, thisNode);
-		setNationalProviderIdOnNode(element, thisNode);
+		if (ENTITY_INDIVIDUAL.equals(thisNode.getValue(ENTITY_TYPE))) {
```
Comment (sfradkin) [inline converter/src/main/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoder.java]: Does this work correctly for CPC+?

File: converter/src/test/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoderTest.java [hunk 2]
```diff
@@ -195,7 +194,7 @@ void decodeClinicalDocumentInternalDecodeMIPSGroup() {
 				.isEqualTo(ClinicalDocumentDecoder.ENTITY_GROUP);
 		assertWithMessage("Clinical Document doesn't contain national provider")
 				.that(testParentNode.getValue(ClinicalDocumentDecoder.NATIONAL_PROVIDER_IDENTIFIER))
-				.isEqualTo("2567891421");
+				.isEqualTo(null);
```
Comment (omnivion) [inline converter/src/test/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoderTest.java]: isNull should be a thing

Comment (clydet) [reply converter/src/test/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoderTest.java]: ah

File: converter/src/main/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoder.java (hunk 1 above)
Comment (clydet) [reply converter/src/main/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoder.java]: tests are passing and CPC+ submissions are regarded as individual.

```
	private String[] getProgramNameEntityPair(String name) {
		String[] pairs;
		if (MIPS_INDIVIDUAL.equalsIgnoreCase(name)) {
			pairs = new String[] {MIPS_PROGRAM_NAME, ENTITY_INDIVIDUAL};
		} else if (MIPS_GROUP.equalsIgnoreCase(name)) {
			pairs = new String[] {MIPS_PROGRAM_NAME, ENTITY_GROUP};
		} else if (CPCPLUS.equalsIgnoreCase(name)) {
			pairs = new String[] {CPCPLUS_PROGRAM_NAME, ENTITY_INDIVIDUAL};
		} else {
			pairs = new String[] {name.toLowerCase(Locale.ENGLISH), ENTITY_INDIVIDUAL};
		}
		return pairs;
	}
```

Comment (sfradkin) [reply converter/src/main/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoder.java]: Ah right.  All good.

**Pull request merged**